languages/csharp/exercises/concept/nullability/.docs/instructions.md

```diff
@@ -0,0 +1,39 @@
+In this exercise you'll be writing code to helping printing name
+badges for factory employees.
+
+Employees have an id, a name and a department name.  For example, if
+my id is `734`, my name is `Ernest Johnny Payne`, and my department is
+`Strategic Communication` my badge name is: `[734] - Ernest Johnny
+Payne - STRATEGIC COMMUNICATION`.
+
+Your task will be writing a static method called `Badge.Label`. It
+will take as input parameters the id, name and department of an
+employee. You will then compute the string to be printed on the
+badge. 
+
+Notice that you'll need to include dashes between words when
+applicable and write the department in UPPER CASE.
```

Comment:
I think you can safely remove this as the above suggestion just specifies the format.